apps/codecov-api/webhook_handlers/views/sentry.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+from django.db import transaction
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from shared.metrics import Counter
+from webhook_handlers.constants import GitHubWebhookEvents
+from webhook_handlers.permissions import JWTAuthenticationPermission
+from webhook_handlers.views.github import GithubWebhookHandler
+
+sentry_webhook = Counter(
+    "sentry_webhook", "Number of sentry webhooks received", ["event"]
+)
+
+log = logging.getLogger(__name__)
+
+
+class SentryWebhookHandler(APIView):
+    permission_classes = [JWTAuthenticationPermission]
+
+    def post(self, request):
+        with transaction.atomic():
+            # this looks weird but we're basically doing "dry runs"
+            # of the webhook handling
+
+            # this will eventually be removed once we validate it's working
+            # correctly in prod
+            transaction.set_rollback(True)
```

Comment:
lets hope the assumption that we can use transaction rollback for this actually works out, lol